desktop-src/direct3dhlsl/waveactivecountbits.md

```diff
@@ -58,7 +58,7 @@ This function is supported from shader model 6.0 in all shader stages.
 This can be implemented more efficiently than a full WaveActiveSum, as described in the following example:
```

Comment:
Maybe this line should be clarified that this is meant to demonstrate what this operation does:
```suggestion
This can be implemented more efficiently than a full WaveActiveSum, in a way similar to this equivalent code:
```

Reply:
Tried another way to put this, hopefully this is clearer?